Loop/Managers/LoopDataManager.swift

```diff
@@ -39,6 +39,14 @@ final class LoopDataManager {
             notify(forChange: .preferences)
         }
     }
+    
+    var batteryIsLithium: Bool {
```

Comment:
Since this variable isn't related to Loop calculations, it probably makes more sense to have it on DeviceDataManager.